frb_rust/src/lib.rs

```diff
@@ -27,6 +27,6 @@ where
 
 /// Marker trait for types that are safe to share with Dart and can be dropped
 /// safely in case of a panic.
-pub trait DartSafe: Send + Sync + UnwindSafe + RefUnwindSafe {}
+pub trait DartSafe: UnwindSafe + RefUnwindSafe {}
```

Comment:
Aha, DartSafe no longer needs to be Send and Sync? Curious to know more details.

Btw, the doc may need some updates as well, since IIRC in somewhere it mentions Send/Sync

Reply:
about `Sync`
`Arc` will also check for the need

```
impl<T> Send for Arc<T>
where
    T: Sync + Send + ?Sized,
 ```

Reply:
I see. So it indeed is "though DartSafe does not explicitly say Send/Sync, it is required if you want to use it in async". Then maybe a brief line of comment would be great